contrib/ubuntu/jammy_ubuntugis_unstable/Dockerfile

```diff
@@ -0,0 +1,64 @@
+#
+# This creates an Ubuntu derived base image to build
+# Kealib and package it as .deb
+# 
+# NOTE: The built package is designed to work on ubuntu jammy
+#
+# Ubuntu 22.04 Jammy Jellyfish
+FROM ubuntu:jammy
+
+MAINTAINER EnvSys <developers@envsys.co.uk>
+
+ENV ROOTDIR /usr/local/
+
+ENV TZ=Europe/London
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Load assets
+WORKDIR $ROOTDIR/
+
+# Install basic dependencies
+RUN apt-get update -y && \
+    apt-get install -y --no-install-recommends software-properties-common gnupg && \
+    add-apt-repository universe && \
+    add-apt-repository -y ppa:ubuntugis/ubuntugis-unstable && \
+    apt-get update -y && apt-get install --no-install-recommends -y \
+    build-essential \
+    libgdal-dev \
+    libhdf5-dev \
+    cmake
+
+ARG KEALIB_VER=1.5.1
+ARG PACKAGE_DIR=/usr/local/packages
+ARG KEALIB_PACKAGE_DIR=$PACKAGE_DIR/libkea_$KEALIB_VER-1
+ARG make_threads=7
+
+RUN mkdir /usr/local/src/kealib
+COPY . /usr/local/src/kealib
+
+RUN mkdir -p $KEALIB_PACKAGE_DIR/DEBIAN && mkdir -p $KEALIB_PACKAGE_DIR/etc/profile.d
+
+# Install kealib
+RUN cd /usr/local/src/kealib && \
+    mkdir build && cd build && \
+    cmake \
+    -D CMAKE_VERBOSE_MAKEFILE=ON \
+    -D BUILD_SHARED_LIBS=ON \
+    -D CMAKE_BUILD_TYPE=Release \
+    -D CMAKE_INSTALL_PREFIX=/usr \
+    .. && \
+    make -j "${make_threads}" && \
+    make install DESTDIR=$KEALIB_PACKAGE_DIR && \
+    cmake \
```

Comment:
Do you need to invoke `cmake` twice, or can building of the gdal plugin be done at the same time?

Reply:
I couldn't get it to work if I tried to build the GDAL plugin the first time.  I don't have the exact error in front of me, but I think it was complaining about not being able to find libkea.so, or something similar.

This led me to believe that I needed to build the kealib shared library first, then the plugin.  I'm fairly sure the wiki implies something of the sort too, though I may well have just made that up.

Reply:
Thanks @maxfenv I've created an issue for this (https://github.com/ubarsc/kealib/issues/36) and will check it out. `cmake` isn't my strong point, I have a bit of trouble working it all out...